db.go

```diff
@@ -36,6 +36,14 @@ import (
 	"github.com/polarsignals/frostdb/wal"
 )
 
+const (
+	B  = 1
+	KB = 1024 * B
```

Comment:
I think this is a great change,  could use change it to use the units here of KiB, MiB etc.

Reply:
Sure. Updated.